.env

```diff
@@ -0,0 +1,16 @@
+PROV_APP_USER=openshift
+PROV_APP_PASSWORD=openshift
+ATLASSIAN_USER=openshift
+ATLASSIAN_PASSWORD=openshift
+JENKINS_USER=developer
+JENKINS_PASSWORD=any
+OPENSHIFT_USER=developer
+OPENSHIFT_PASSWORD=any
+PROV_APP_NAME=prov-app-ods
+JIRA_URL=http://jira.odsbox.lan:8080/
+OPENSHIFT_PROJECT=edpp
+OPENSHIFT_PUBLIC_HOST=ocp.odsbox.lan
+OPENSHIFT_CLUSTER=https://ocp.odsbox.lan:8443/
+BITBUCKET_URL=http://bitbucket.odsbox.lan:7990
+SIMULATE=false
+
```

Comment:
should this .env file be committed? .env.example would be a better choice with the example/undefined values

Reply:
or .env.template :)

Reply:
if @jorge-romero  - sources the env (in case it exists) - those should be overwritten by what's in the sourced env.

Reply:
@m-apsolon  .- agreed! . I would call it .sample .. and add the .env to gitignore

Reply:
It can be renamed as you suggest and excluded in the .gitignore so no one will accidentally commit the credentials / routes of the environment.

Reply:
Done :)

Reply:
Added code to load and override the environment setup from:

1) ods-configuration/ods-core.env
2) .env -> It should contain the passwords and other needed information. I can override the ods-core.env environment or it will be the first source if the tests are executed in a computer that does not have the ods-configuration folder.
3) The environment variables, that will override the previous ones as the last place to setup the tests.